spec/active_hash/base_spec.rb

```diff
@@ -116,6 +116,16 @@ class Country < ActiveHash::Base
     end
   end
 
+  describe ".column_names" do
+    before do
+      Country.fields :name, :iso_name, "size"
+    end
+
+    it "returns an array of column names" do
+      expect(Country.column_names).to eq(["name", "iso_name", "size"])
+    end
```

Comment:
@flavorjones do we want to just conditionally do this?

```suggestion
    if RUBY_VERSION >= "3.0"
    it "returns an array of column names" do
      expect(Country.column_names).to eq(["name", "iso_name", "size"])
    end
    end
```

Reply:
Seems fine to me.

Reply:
I commited! https://github.com/active-hash/active_hash/pull/311/commits/9679a8998128c2e11c994e76dde132135759f153